extensions/npm/package.json

```diff
@@ -88,6 +88,10 @@
       {
```

Comment:
Currently the command is also shown in the command palette

![image](https://user-images.githubusercontent.com/172399/66516853-4903b900-eae2-11e9-98c2-f497f79c77b9.png)

The command assumes to receive a `path` as the first argument and it will therefore fail.
![image](https://user-images.githubusercontent.com/172399/66516920-718bb300-eae2-11e9-862a-dd9bb9a0e5b6.png)

The command should either not be shown in the command palette or handle the case that it doesn't receive a `path` as argument. To not show a command in the command palette can be done as follows:
```json
    "menus": {
      "commandPalette": [
        {
          "command": "npm.refresh",
          "when": "false"
        },
```